src/vm/jitinterface.cpp

```diff
@@ -538,6 +538,15 @@ CEEInfo::ConvToJitSig(
         ULONG data;
         IfFailThrow(sig.GetCallingConvInfo(&data));
         sigRet->callConv = (CorInfoCallConv) data;
+
+#if defined(FEATURE_CORECLR)
+        if (sigRet->callConv & IMAGE_CEE_CS_CALLCONV_VARARG)
```

Comment:
@jkotas The comment says it is used for 64bit PInvokes only but I dont see it being documented or used in that manner. Is it really only used for 64bit PInvoke?